src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpectatorView.cs

```diff
@@ -244,6 +270,59 @@ private bool TryCreateRecordingService(out IRecordingService recordingService)
 #endif
         }
 
+        private void SetupNetworkConfigurationVisual()

```

Comment:
>SetupNetworkConfigurationVisual [](start = 21, length = 31)

Do you think the network connection visual should re-show when a disconnect happens? Should there be additional UI or a public method that an app can call to re-show the connection UI?

Reply:
This seems like a cool improvement. Maybe we could address this in a separate review? Users may also want to dynamically change the device that they are spectating as well.

Reply:
opened https://github.com/microsoft/MixedReality-SpectatorView/issues/133 for tracking